### PR TITLE
energy 2.4.6 (new formula)

### PR DESCRIPTION
**Pull request by sxmxta**
Add Energy CLI development environment tool

Energy CLI is a development environment for energy frameworks It can be installed, compiled, and packaged from the development environment

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by sxmxta** [comment]
got it

**Comment by sxmxta** [comment]
I am not very familiar with the operations on Git (GitHub), so I keep creating PRs repeatedly. I apologize for the inconvenience.

File: Formula/e/energy.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Energy < Formula
+  desc "CLI is used to initialize the Energy development environment tools"
+  homepage "https://energye.github.io"
+  url "https://github.com/energye/energy/archive/refs/tags/v2.4.5.tar.gz"
```
Comment (daeho-ro) [inline Formula/e/energy.rb:4]: The version of tarball is 2.4.5 and version of cli which can be obtained from the static file of homepage is 1.0.10. I think then the version of formula will be 2.4.5 that is not match with cli binary.

Comment (sxmxta) [reply Formula/e/energy.rb]: The mismatch between the CLI version number and Formula has been resolved

**Review by daeho-ro — DISMISSED** [review]
It looks fine. Thank you!

**Comment by sxmxta** [comment]
Hello.
Is there anything else I need to do next?

**Review by SMillerDev — DISMISSED** [review]
Thanks @sxmxta  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request merged**